Week-3/Bootstrap for a vacation website/script.js

```diff
@@ -0,0 +1,12 @@
+function showAlert(alertId) {
+    const bootstrapAlert = document.querySelector("#" + alertId);
+    const collapse = new bootstrap.Collapse(bootstrapAlert); 
+    collapse.show();
+ }
+ 
+ const modalYesBtn = document.querySelector("#contact-modal .btn-primary");
+ modalYesBtn.addEventListener("click", function () {    
+    const sendBtn = document.querySelector("#send-message-btn");
```

Comment:
query selector is slow, i see you are accessing id only 
use document.getElementById 

getElementById is way faster